R/error-checks.R

```diff
@@ -120,7 +120,7 @@ isFileExtension <- function(file, extension) {
   file_ext %in% extension
 }
 
-#' Check that an a array of values does not include any duplicate
+#' Check that an array of values does not include any duplicate

```

Comment:
This change also needs to be reflected in `.Rd` file.

Reply:
@pchelle Can you update the documentation so that we can close this PR?

Reply:
Let's merge this for now. I can change this in the `main` branch itself.